src/store.ts

```diff
@@ -39,41 +39,50 @@ const dbAllAsync = function(db: sqlite3.Database, sql: string) {
   })
 }
 
-export function init() {
-  return new Promise((resolve, reject) => {
-    if (!db) {
-      db = new sqlite3.Database(isTesting ? ':memory:' : dbPath, err => {
-        if (err) {
-          return reject(err)
-        }
-
-        resolve()
+export async function init(resetDB = false) {
+  return new Promise(async (resolve, reject) => {
+    if (db) {
+      await new Promise(resolve => {
+        db.close(() => {
+          if (fs.existsSync(dbPath)) {
+            fs.unlinkSync(dbPath)
+          }
+          resolve()
+        })
       })
     }
+
+    if (resetDB) {
+      if (fs.existsSync(dbPath)) {
+        fs.unlinkSync(dbPath)
+      }
+
+      // remove all ballot images and scanned ballot images
```

Comment:
nit: Do it or TODO it!

Reply:
oops, it's actually being done in `scanner.ts`, I forgot to remove this comment, removing now.